README.md

```diff
@@ -15,7 +15,6 @@ You are viewing the `3.x` documentation. [Click here](https://github.com/laravel
 ## Contents
 
 - [Installation](#installation)
-	- [Setting up your Twilio account](#setting-up-your-twilio-account)
```

Comment:
Should this be linked to twilio if there isn't going to be anything about it in the readme?